CharlieBackend.Business/Helpers/PasswordHelper.cs

```diff
@@ -57,5 +58,47 @@ private static Int32 Next(Int32 minValue, Int32 maxValue)
                 }
             }
         }
+
+        public static string PasswordValidation(string password)
```

Comment:
Do we really need to return strings with detailed info on why password is invalid? I mean, it's fine if we do. 
But I feel like this validation could be simplified. We could return `bool` (password is either valid or not). Method could be called `PasswordIsValid()`. And we could get rid of all of this `if` statements.
Again, just a suggestion.